lib/web/apiserver.go

```diff
@@ -436,6 +436,8 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*RewritingHandler, error) {
 			ctx, err := h.AuthenticateRequest(w, r, false)
 			if err == nil {
 				resp, err := newSessionResponse(ctx)
+				resp.WebIdleTimeout = int(cfg.WebIdleTimeout.Milliseconds())
```

Comment:
make `newSessionResponse` part of the `Handler` so you can access `cfg` from that function. This will ensure that `sessionInactivityTimeout` always gets passed (even during renews).

```golang
func (h *Handler) newSessionResponse(ctx *SessionContext) (*CreateSessionResponse, error) {
}
```

Reply:
revised